__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require(`../app/app`)
+const request = require('supertest')
+const express = require('express')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+
+app.use(express.json())
+
+beforeEach(()=>{
+    return seed(data)
+})
+
+afterAll(()=>{
+    db.end()
+})
+
+describe("GET requests", ()=>{
+    test("GET /api/topics - should return an array of objects", ()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body})=>{
+            body.forEach((topic)=>{
+                expect(topic).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                })
+            })
+        })
+    })
```

Comment:
This is a good start to this test, but body should be an object with an appropriate key that holds the array of topics.

Currently this test would still pass if the topics array was an empty array, can you think of a way to ensure that the array must contain some elements within it?